### DISCUSSION TITLE
Ratio and transfer shown together?

### DISCUSSION THREAD

**Discussion by edaragor** [transmission/transmission#6776] [Ideas]
Hi, love the app, using it on OSX <3

It would be great if it was possible to show the ratio together with transfer amount.
So instead of only these 4 options, then two options more like:

- Total Transfer & Total Ratio
- Session Transfer & Session Ratio

![Screenshot 2024-04-07 at 19 16 52](https://github.com/transmission/transmission/assets/36200576/8e459f0d-73fd-4bda-a457-04b3963a7572)

**Comment by Pentaphon** [comment]
>So instead of only these 4 options, then two options more like:

Or you can just make 6 total options so nobody loses their option. 

You should make an issue for this here: https://github.com/transmission/transmission/issues

**Reply by killemov** [reply]
I severely dislike the way issues and discussions are haphazardly transformed in to each other here. For some issues you now also need to read the discussion to know what's going on and sometimes even the other way around.
